lib/ofx/parser/ofx220.rb

```diff
@@ -7,6 +7,10 @@ def accounts_info
         @accounts_info ||= html.search('acctinfo').collect { |node| build_account_info(node) }
       end
 
+      def credit_card_closing_info
```

Comment:
This is present in the OFX 1.0.2 specs so you should put this method in the 1.0.2 parser